universal-application-tool-0.0.1/app/services/applicant/question/Scalar.java

```diff
@@ -27,7 +27,8 @@
   LINE2("address line 2", ScalarType.STRING),
   MIDDLE_NAME("middle name", ScalarType.STRING),
   NUMBER("number", ScalarType.LONG),
-  SELECTION("selection", ScalarType.LIST_OF_STRINGS),
+  // Selection may be either STRING (single-select) or LIST_OF_STRINGS (multi-select)
+  SELECTION("selection", ScalarType.LIST_OF_STRINGS, ScalarType.STRING),
```

Comment:
The previous addition of `ScalarType` as part of a `Scalar` needs some clean up. It is very confusing that there is a (scalar, scalar type) pair here, and then a different set of pairs down below in things like `ADDRESS_SCALARS`.

Reply:
Oh yes @kendrickt thank you for the reminder. I've been meaning to clean that up.

Reply:
https://github.com/seattle-uat/civiform/pull/1429